app/Controllers/Http/ProfileController.ts

```diff
@@ -0,0 +1,76 @@
+// import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from "App/Models/Profile"
+import User from "App/Models/User"
+import UserProfileValidator from "App/Validators/UserProfileValidator"
+import {DateTime} from "luxon"
+
+export default class ProfileController {
+
+    public async view(){
+        const profiles=await Profile.all()
+        return profiles
+    }
+
+    public async showProfile({auth}){
+        
+        try{
+            const profile=await Profile.findBy("user_id",auth.user.$attributes.id)
+            const details={}
+
+            const dob=profile.$attributes.dob.c.year+"-"+profile.$attributes.dob.c.month+"-"+profile.$attributes.dob.c.day
```

Comment:
we can avoid using `$attributes` node properties can be access directly  eg: `profile.dob`